toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -50,6 +50,11 @@ const (
 	// /boot directory should be only accesible by root. The directories need the execute bit as well.
 	bootDirectoryFileMode = 0400
 	bootDirectoryDirMode  = 0700
+
+	CmdlineSELinuxSettings       = "security=selinux selinux=1"
+	CmdlineSELinuxForceEnforcing = CmdlineSELinuxSettings + " enforcing=1"
+
+	SELinuxConfigFile = "etc/selinux/config"
```

Comment:
Small ask: since we're here, could we split the `const` sections into two? One for public constants and one for private ones? We tried to do it in many other places in the code and it usually makes it easier to distinguish the what is internal-only and what is intended to be used by other packages.

Reply:
fixed